### PR TITLE
Fix rules that disallow passive-aggressive nomination

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/ice-wg/rfc5245bis/issues/18
Also addresses Issue https://github.com/ice-wg/rfc5245bis/issues/15

This PR does not change the example in Section 12 (is this necessary if the text was clear enough?).

### CODE REVIEW

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -3171,56 +3171,28 @@ Procedures for sending media differ for full and lite implementations.
...
+stream. Once that happens, the agent MAY begin sending media
+packets.
+</t>
+	
+<t>
+An agent will send media to the remote candidate (setting the 
+destination address and port of the packet equal to that remote candidate), 
+and will send it from the local candidate. When the local candidate is
 server or peer reflexive, media is originated from the base. Media
 sent from a relayed candidate is sent from the base through that TURN
```
Comment (cdh4u) [inline draft-ietf-ice-rfc5245bis.xml:3185]: Not really related to aggressive nomination, but is there a reason we can't say that media is always sent from the base, no matter the candidate type?

Comment (aboba) [reply draft-ietf-ice-rfc5245bis.xml:3185]: This PR only proposes changes directly related to passive-aggressive nomination.  The text relating to sending media has not been changed.  So your concern seems orthogonal and would be best handled in a separate issue/PR.

**Comment by aboba** [comment]
@taylor-b  Can you review?

File: draft-ietf-ice-rfc5245bis.xml [hunk 2]
```diff
@@ -3171,56 +3171,28 @@ Procedures for sending media differ for full and lite implementations.
...
 <t>
-Agents always send media using a candidate pair, called the selected
-candidate pair. An agent will send media to the remote candidate in
-the selected pair (setting the destination address and port of the
-packet equal to that remote candidate), and will send it from the
-local candidate of the selected pair. When the local candidate is
+A full implementation MUST NOT send media until it has a Valid list
+that contains a candidate pair for each component of that media
+stream. Once that happens, the agent MAY begin sending media
+packets.
```
Comment (taylor-b) [inline draft-ietf-ice-rfc5245bis.xml:3177]: May want to clarify that the agent may only begin sending media packets "using candidate pairs in the Valid list".

File: draft-ietf-ice-rfc5245bis.xml [hunk 3]
```diff
@@ -3171,56 +3171,28 @@ Procedures for sending media differ for full and lite implementations.
...
-<t>
-If the selected pair for at least one component of a media stream is
-empty, an agent MUST NOT send media for any component of that media
-stream. If the selected pair for each component of a media stream has
-a value, an agent MAY send media for all components of that media
-stream.
-</t>
-
-
-
```
Comment (taylor-b) [inline draft-ietf-ice-rfc5245bis.xml:3223]: Is it right to delete all this? I assumed that once a pair _is_ selected, the agent MUST use it. Meaning we'd keep this, but relax the "MUST NOT" above.

**Review by taylor-b — COMMENTED** [review]
Thanks for writing the PR, it's been on my todo list for a while.

File: draft-ietf-ice-rfc5245bis.xml (hunk 3 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml:3223]: Any news regarding Taylor's comment?

**Pull request closed without merging**